### PR TITLE
Replace getGraphQlContext with createHTTPContext

### PR DESCRIPTION
**Pull request by timleslie**

### CODE REVIEW

**Comment by gautamsi** [comment]
Is this aligned with removal of executeQuery? Should this be staged for later?

**Comment by timleslie** [comment]
> Is this aligned with removal of executeQuery? Should this be staged for later?

Yes, that's why this PR is still in `[WIP]`.

File: packages/keystone/lib/Keystone/index.js [hunk 1]
```diff
@@ -263,17 +263,16 @@ module.exports = class Keystone {
...
-  // It is also used for generating the `keystone.query` method
-  getGraphQlContext({ schemaName, req = {}, skipAccessControl = false } = {}) {
+  createHTTPContext({ schemaName, req }) {
+    // The GraphQL App uses this method to build up the context required for each incoming query.
     return {
       ...this.createContext({
         schemaName,
         authentication: { item: req.user, listKey: req.authedListKey },
-        skipAccessControl,
+        skipAccessControl: false,
```
Comment (gautamsi) [inline packages/keystone/lib/Keystone/index.js:272]: is this going to break use when want to skip access control

Comment (MadeByMike) [reply packages/keystone/lib/Keystone/index.js:277]: That's a default value so if you create a context with `skipAccessControl: true`, no prob.

File: packages/keystone/lib/Keystone/index.js [hunk 2]
```diff
@@ -234,17 +234,16 @@ module.exports = class Keystone {
     return execute(schema, query, null, context, variables);
   }
 
-  // The GraphQL App uses this method to build up the context required for each
-  // incoming query.
-  // It is also used for generating the `keystone.query` method
-  getGraphQlContext({ schemaName, req = {}, skipAccessControl = false } = {}) {
+  createHTTPContext({ schemaName, req }) {
```
Comment (jordanoverbye) [inline packages/keystone/lib/Keystone/index.js:237]: Do we need `req = {}` here?

Comment (timleslie) [reply packages/keystone/lib/Keystone/index.js:237]: No, `req` is a required argument for this use case, which is specifically for HTTP requests.

**Pull request merged**